code/__DEFINES/economy.dm

```diff
@@ -3,11 +3,16 @@
 #define PAYCHECK_ASSISTANT 5
 #define PAYCHECK_MINIMAL 5
 #define PAYCHECK_EASY 20
-#define PAYCHECK_MEDIUM 50
-#define PAYCHECK_HARD 100
-#define PAYCHECK_COMMAND 200
+#define PAYCHECK_MEDIUM 30
+#define PAYCHECK_HARD 40
+#define PAYCHECK_COMMAND 100
 
-#define PAYCHECK_WELFARE 20 //NEETbucks
+#define PAYCHECK_WELFARE 5 //NEETbucks
```

Comment:
might work irl but a twinkie costs 11$ in game

Reply:
That sounds like more of an issue on the base pay because having the neet perk shouldnt be a requirement to eat, especially since it already would be the by far the best 1 cost perk if they didnt get paid at all from it

Reply:
If I subsisted entirely on twinkies from vending machines in real life I too would be poor, fat and unemployed